packages/main/src/Menu.ts

```diff
@@ -358,11 +359,13 @@ class Menu extends UI5Element {
 	}
 
 	get isSubMenuOpened() {
-		return this._parentMenuItem && this._popover?.isOpen();
+		return this.items.some(menuItem => menuItem.subMenuOpened);
 	}
 
 	get menuHeaderTextPhone() {
-		return this._parentMenuItem ? this._parentMenuItem.text : this.headerText;
+		const menu = this as Menu;
```

Comment:
This cast can be removed. This is a menu

Reply:
done